src/main/java/mate/hwdao/dao/impl/jdbc/CarDaoJdbcImpl.java

```diff
@@ -0,0 +1,197 @@
+                .prepareStatement(query)) {
+            preparedStatement.setLong(1, id);
+            return preparedStatement.executeUpdate() > 0;
+        } catch (SQLException ex) {
+            throw new DataProcessingException("cars with " + id
+                    + "was not deleted because of", ex);
+        }
+    }
+
+    public List<Car> getAllByDriver(Long driverId) {
+        String query = "Select cars.id, manufacturer_id, model, deleted, "
+                + "manufactures.id, manufactures.name, manufactures.country "
+                + "From cars  INNER JOIN manufactures ON cars.manufacturer_id = manufactures.id "
+                + "JOIN cars_drivers WHERE cars.id = cars_drivers.car_id "
+                + "AND  cars_drivers.driver_id = ? AND deleted = false";
+        try (PreparedStatement preparedStatement
+                     = ConnectionUtil.getConnection()
+                .prepareStatement(query)) {
+            preparedStatement.setLong(1, driverId);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            List<Car> cars = new ArrayList<>();
+            while (resultSet.next()) {
+                cars.add(makeCar(resultSet));
+            }
+            return cars;
+        } catch (SQLException ex) {
+            throw new DataProcessingException("Can't get list of car by driver id: "
+                    + driverId, ex);
+        }
+    }
+
+    private void delateOldDrivers(String query, Car car) {
+        try (PreparedStatement deletedOldDriversStatement = ConnectionUtil.getConnection()
+                .prepareStatement(query)) {
+            deletedOldDriversStatement.setLong(1, car.getId());
+            deletedOldDriversStatement.executeUpdate();
+        } catch (SQLException ex) {
+            throw new DataProcessingException("Can't save " + car
+                    + " failed", ex);
+        }
+    }
+
+    private void setNewDriversStatement(String query, Car car) {
+        try (PreparedStatement setNewDriversStatement = ConnectionUtil.getConnection()
+                .prepareStatement(query)) {
+            for (Driver driver : car.getDrivers()) {
+                setNewDriversStatement.setLong(1, driver.getId());
+                setNewDriversStatement.setLong(2, car.getId());
+                setNewDriversStatement.executeUpdate();
+            }
+        } catch (SQLException ex) {
+            throw new DataProcessingException("Can't save " + car
+                    + " failed", ex);
+        }
+    }
+
+    private Car makeCar(ResultSet resultSet) throws SQLException {
+        Car car = new Car();
+        car.setId(resultSet.getObject(1, Long.class));
+        car.setModel(resultSet.getObject(3, String.class));
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setId(resultSet.getObject(5, Long.class));
+        manufacturer.setName(resultSet.getObject(6, String.class));
+        manufacturer.setCountry(resultSet.getObject(7, String.class));
```

Comment:
I agree  - let's use column names instead of numbers

Reply:
done